src/setuptools_scm/_integration/dump_version.py

```diff
@@ -15,9 +15,13 @@
     ".py": """\
 # file generated by setuptools-scm
 # don't change, don't track in version control
+
+__all__ = ["__version__", "__version_tuple__", "version", "version_tuple"]
+
 TYPE_CHECKING = False
 if TYPE_CHECKING:
     from typing import Tuple, Union
+
```

Comment:
This would be done by `ruff` format if you ran it on the file, so I included it

Reply:
https://github.com/pypa/setuptools-scm/blob/d665621e25f091c0de5cd35b3d57cc00bd26e260/testing/test_functions.py#L136-L153

needs to add ruff as followup